### PR TITLE
refactor helm v3 script to simplify k8s+s3 deploy

### PR DESCRIPTION
**Pull request by weii666**
No git cloning or yaml downloading is required for this easy installation.
Here are the installation steps after this change.

1. export 4 environment variables for S3.
2. run installExporterChart without any argument

I have tested on my OpenShift cluster and IBM S3 object store.
Data objects are collected every 5 minutes per nodes.

Temporal Helm chart repo for reference:
https://github.com/weii666/charts

Thank you for review.

### CODE REVIEW

File: helm/helm-scpts-v3/installExporterChart [hunk 1]
```diff
@@ -46,23 +43,16 @@ do
...
+    echo "Warning: Namespace 'sysflow' already exists. Proceeding with existing namespace"
 else
     echo "Namespace 'sysflow' created successfully"
 fi
 
-helm install sf-exporter-chart -f sf-exporter-chart/values.yaml.local --namespace sysflow --set sfexporter.s3AccessKey=$s3AccessKey --set sfexporter.s3SecretKey=$s3SecretKey --set sfexporter.s3Endpoint=$s3Endpoint --set sfexporter.s3Location=$s3Region --debug ./sf-exporter-chart
+# TODO: publish sysflow charts to https://kubernetes-charts.storage.googleapis.com,
+#       which is enabled by default, and get rid of below version and repo strings.
+chartVersion="0.1-rc2"
+chartRepo=https://weii666.github.io/charts/
```
Comment (araujof) [inline helm/helm-scpts-v3/installExporterChart:56]: @weii666 did you mean to hardcode your personal repo here? Can you please fix this with the sf-deployments charts URL?

**Review by araujof — CHANGES_REQUESTED** [review]
Hi @weii666, can you please address my comment on helm/helm-scpts-v3/installExporterChart and resubmit? Once you send us an updated PR, I will approve it quickly. Thanks!

File: helm/helm-scpts-v3/installExporterChart (hunk 1 above)
Comment (weii666) [reply helm/helm-scpts-v3/installExporterChart:56]: This URL must be a Helm charts repo, where is index.yaml located.
My github is configured as a web page for this purpose.
We can make /sf-deployments/ be the Helm charts repo.
It needs to enable GitHub Pages in repo setting.

The correct way is to publish sysflow charts on https://kubernetes-charts.storage.googleapis.com as the TODO states above.
I can help to publish it.
Thank you.

Comment (zvn) [reply helm/helm-scpts-v3/installExporterChart:56]: if we are using Operator installation on k8s, and let operator points to the charts served by ourselves, do we still need this helm-chart based installation?

Comment (terylt) [reply helm/helm-scpts-v3/installExporterChart:56]: We still want to support vanilla k8's no?

Comment (weii666) [reply helm/helm-scpts-v3/installExporterChart:56]: Publishing to helm repo is for community users.
It provides an option to deploy sysflow without operator.
The PR sent to helm is under review and has not got response.
I will try to ping helm members to continue.

Comment (weii666) [reply helm/helm-scpts-v3/installExporterChart:56]: I got e2e test result from helm community.
Here is an error installing Sysdig:

I0414 12:19:07.157] Your kernel headers for kernel 4.19.104+ cannot be found at
......
I0414 12:19:07.158] E0414 12:18:59.790402    10 main.cpp:233] Runtime exception caught in main loop: error opening device /host/dev/sysdig0. Make sure you have root credentials and that the sysdig-probe module is loaded.

We do require privilege to mount /dev/ for sf-collector, 
which is not a normal behavior to any automatic tests.
I suspect we will have the same issue to pass RedHat certification.

Comment (terylt) [reply helm/helm-scpts-v3/installExporterChart:56]: Looks like the issue is that you don't have the kernel headers installed on the host.

Comment (weii666) [reply helm/helm-scpts-v3/installExporterChart:56]: Here is the helm PR and test links are within:
https://github.com/helm/charts/pull/21730

**Pull request closed without merging**